### PR TITLE
Populate FINANCES.md

### PR DESCRIPTION
**Pull request by tcr**
Subject to review, please!

### CODE REVIEW

**Comment by johnnyman727** [comment]
r+

File: FINANCES.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
-Division of finances for funding tools, R&D, and bug/feature bounties.
+The Tessel Project (as of May 2016) has its funds held by the non-profit Dojo Foundation.
 
----
+Funds for the Tessel Project are generated in a few ways:
 
-TODO
+1. We receive royalties from our manufacturing partners, usually a few percentage points of each product sold.
+2. We generated money through Amazon referral links for hardware, for example [our hardware modules](http://github.com/tessel/hardware-modules) recommendation list.
+3. We receive money directly through donations.
```
Comment (Frijol) [inline FINANCES.md]: ..we do?

File: FINANCES.md [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+4. We receive money through partnerships or sponsorships with companies.
+
+## What these funds are used for
+
+The Tessel Project has no persons on payroll and is staffed entirely by volunteers. Funds must be approved for use by the Steering Committe. These are allocated for a few broad uses:
```
Comment (Frijol) [inline FINANCES.md]: typo: Committe -> Committee

File: FINANCES.md [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
+
+1. Server hosting and domain names.
+1. The purchasing of hardware directly from manufacturing partners for use in hackathons, advertisements, conferences, or direct engagement with customers or team members.
```
Comment (Frijol) [inline FINANCES.md]: strike leading "the" for list consistency

File: FINANCES.md [hunk 4]
```diff
@@ -1,7 +1,25 @@
...
+2. Equipment for the testing and validation of hardware. For example: micro USB cables, Tessels, and Tessel modules may be purchased to diagnose a specific hardware issue occurring under circumstances we need to recreate in-house. Another example is the purchase of logic or radio analyzers, which are usually shared within the team.
+3. In limited (one-off) quantities, we may purchase swag to reward volunteers.
```
Comment (Frijol) [inline FINANCES.md]: volunteers -> contributors ?

**Comment by Frijol** [comment]
Yay, thank you for doing this!

Might also be useful to add a note re guiding philosophy of frugality.
Or not.

**Comment by Frijol** [comment]
I made a PR to this PR: https://github.com/tessel/project/pull/167

**Comment by Frijol** [comment]
ping @tcr

File: FINANCES.md (hunk 1 above)
Comment (Frijol) [inline FINANCES.md]: Well, we do now (https://opencollective.com/tessel)

**Pull request merged**